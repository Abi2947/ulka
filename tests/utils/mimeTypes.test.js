const mimeType = require('../../src/utils/mimeTypes')

describe('mimeType', () => {
  test('should return mimetype object', () => {
    expect(mimeType).toMatchSnapshot()
  })
})
