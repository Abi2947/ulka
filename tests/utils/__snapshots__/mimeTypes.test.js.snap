// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mimeType should return mimetype object 1`] = `
Object {
  ".css": "text/css",
  ".doc": "application/msword",
  ".eot": "application/vnd.ms-fontobject",
  ".html": "text/html",
  ".ico": "image/x-icon",
  ".jpg": "image/jpeg",
  ".js": "text/javascript",
  ".json": "application/json",
  ".mp3": "audio/mpeg",
  ".pdf": "application/pdf",
  ".png": "image/png",
  ".svg": "image/svg+xml",
  ".ttf": "application/x-font-ttf",
  ".wav": "audio/wav",
  ".zip": "application/zip",
}
`;
